quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -40,11 +40,19 @@
 DEPLOYMENT_NAME = 'quickstart_deployment_' + iso_time
 CONNECTOR_NAME  = 'quickstart_cac_' + iso_time
 
+# Install flags
+INSTALL_TERRAFORM        = False
```

Comment:
why do these have to be global if they are only used within one function?

Reply:
`INSTALL_TERRAFORM` is used towards the end to print the command string back to the user. I guess leaving the fully qualified path regardless of Terraform is installed or not would work too right? But I just thought it may be cleaner for the user if they can just use `terraform destroy` vs `/home/bin/terraform destroy`

Reply:
Global flags have been removed.